tests/atest/rules/misc/variable-not-used/test.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Documentation    This is suite doc
+
+
+*** Test Cases ***
+Test
+    [Documentation]   doc
+    ${var}    Keyword
+
+
+*** Keywords ***
```

Comment:
What about tests for test cases?

Reply:
There is only one cases for `Test Cases` now.
I'll copy the cases in the keywords and adjust them for test cases.